wdl2cwl/tests/wdl_files/CollectReadgroupBamQualityMetrics.wdl

```diff
@@ -0,0 +1,63 @@
+version 1.0
+
+## Copyright Broad Institute, 2018
+##
+## This WDL defines tasks used for QC of human whole-genome or exome sequencing data.
+##
+## Runtime parameters are often optimized for Broad's Google Cloud Platform implementation.
+## For program versions, see docker containers.
+##
+## LICENSING :
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker
+## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
+## licensing information pertaining to the included programs.
+
+task CollectReadgroupBamQualityMetrics {
+  input {
+    File input_bam
+    File input_bam_index
+    String output_bam_prefix
+    File ref_dict
+    File ref_fasta
+    File ref_fasta_index
+    Boolean collect_gc_bias_metrics = true
+    Int preemptible_tries
+  }
+
+  Float ref_size = size(ref_fasta, "GiB") + size(ref_fasta_index, "GiB") + size(ref_dict, "GiB")
+  Int disk_size = ceil(size(input_bam, "GiB") + ref_size) + 20
+
+  command {
+    # These are optionally generated, but need to exist for Cromwell's sake
+    touch ~{output_bam_prefix}.gc_bias.detail_metrics \
```

Comment:
Is it required to convert the  ```touch ~{output_bam_prefix}.gc_bias.detail_metrics \``` commands too?

Reply:
all content in the `command` block should go into the BASH script.